content/en/blog/_posts/Creating AWS Volume Snapshots in Kubernetes Using the EBS CSI Driver

```diff
@@ -0,0 +1,651 @@
+
+`VolumeSnapshot` is a request for a snapshot from the CSI driver (snapshotter), which is similar to how PVCs interact with built-in volume provisioners. 
+
+`VolumeSnapshotContent` is a snapshot of some volume in a K8s cluster. This primitive has a role similar to [PersistentVolumes (PVs)](https://supergiant.io/blog/persistent-storage-with-persistent-volumes-in-kubernetes/). You can use it for the static provisioning of snapshots similarly to how PVs are used for the static provisioning of Kubernetes volumes. If users create a `VolumeSnapshotContent`, it should refer to the real storage available for cluster users. In other words, volumes for `VolumeSnapshotContent` should be pre-provisioned. 
+
+Finally, `VolumeSnapshotClass` is a primitive that describes how snapshots should be taken. A VSC can specify driver information, secrets to pass to a driver, and various driver-specific options. This primitive is quite similar to `StorageClass` because it's used for the dynamic provisioning of volume snapshots without allocating a `VolumeSnapshotContent` beforehand. All these primitives require a CSI driver and snapshot-compatible Kubernetes cluster to work. In general, the following prerequisites should be met:
+
+- Kubernetes >= 1.12.0. Some CSI drivers may require Kubernetes > 1.13.0 or any specific K8s version.
+- Users may have to enable CSI-related and volume snapshot feature gates. At the minimum, you'll need `--feature-gates=VolumeSnapshotDataSource=true` set on the kube-apiserver binary during the cluster provisioning. Using volume snapshotting in the earlier K8s versions may also require enabling CSI-related feature gates. Follow the feature gate status in this [comprehensive doc](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) and consult the documentation of your CSI driver to find out more.
+- A CSI driver for your storage vendor deployed to the cluster.
+
+We'll describe all these requirements in detail using the example of the AWS EBS CSI driver. Let's move on to the tutorial.
+
+### Tutorial
+
+Examples in this tutorial were tested in the following environment:
+
+- Kubernetes 1.14.1 deployed with [Kops 1.14.0-alpha.1](https://github.com/kubernetes/kops/releases/tag/1.14.0-alpha.1) release on AWS
+- the latest release of the [AWS EBS CSI driver](https://github.com/kubernetes-sigs/aws-ebs-csi-driver)
+
+To complete these examples, you'll also need:
+
+- an AWS account with the ability to create IAM users
+- AWS CLI
+- kubectl
+
+In what follows, we'll first quickly provision a K8s cluster on AWS using Kops. The cluster will be configured with all required feature gates.
+
+**NOTE**: all code snippets and manifests used in this tutorial can be obtained from this [GitHub repo](https://github.com/supergiant/blog-code-examples/tree/master/volumesnapshots-aws-ebs-csi). 
```

Comment:
You should be able to use the Hugo syntax for a note in blog entries, I think.
(I haven't tried). That'll give a consistent appearance.